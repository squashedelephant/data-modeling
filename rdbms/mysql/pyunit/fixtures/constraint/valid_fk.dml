INSERT INTO inventory(name, sku, quantity) VALUES ('sandwich', 1001, 10), ('chips', 1002, 5), ('soda', 1003, 3);
INSERT INTO client_order(inv_id, quantity, price) VALUES ((SELECT id FROM inventory WHERE name = 'sandwich'), 1, 6.95);
