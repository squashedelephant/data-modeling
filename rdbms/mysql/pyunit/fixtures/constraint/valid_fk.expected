sandwich
1L
6.95
